### DISCUSSION TITLE
Any way to get this to work when using a src directory?

### DISCUSSION THREAD

**Discussion by scpaye506** [RyanClementsHax/storybook-addon-next#45] [Q&A] [answered]
We are using the src directory feature - https://nextjs.org/docs/advanced-features/src-directory and I'm getting the below error:

```
ModuleNotFoundError: Module not found: Error: Can't resolve '../styles/globals.css'
```

**Comment by RyanClementsHax** [comment, marked as answer]
Thanks for bringing this up. This is actually a bug. Can you open up an issue and provide a reproduction repo?

**Reply by scpaye506** [reply]
Sure - https://github.com/RyanClementsHax/storybook-addon-next/issues/46
